man/common/hierarchy.md

```diff
@@ -0,0 +1,6 @@
+    Suported options are:
+      * **o** for **TPM_RH_OWNER**
+      * **p** for **TPM_RH_PLATFORM**
+      * **e** for **TPM_RH_ENDORSEMENT**
+      * **n** for **TPM_RH_NULL**
+      * **`<num>`** where a raw number can be used.
```

Comment:
maybe **`<hex>`** where an hexadecimal number can be used.

Reply:
You can use octal, or base 10, or even higher bases as described in man strtoul()

Reply:
Ah, right. I forgot we supported any base.